src/index-metadbs/README.md

```diff
@@ -0,0 +1,41 @@
+# Static Index Meta-Databases
+
+## Goal
+
+This folder hosts static versions of Index Meta-Databases for those providers that only have one main sub-database (or very few sub-databases) and do not wish to maintain an Index Meta-Database themselves (Index Meta-Databases are described in the [OPTIMADE specifications](https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst)).
+
+Note: while providing an (empty) Index Meta-Database is not required by the OPTIMADE API specifications, it is instead required in order to be listed in this List of Providers ([providers.optimade.org](http://providers.optimade.org)).
+
+## When should I use a static index meta-database hosted in this repository
+
+If you add a new provider and do not wish to host an Index Meta-Database yourself, feel free to add an Index Meta-Database here.
+
+However, if your Index Meta-Database is rapidly changing (e.g. because the list of sub-databases changes often), please host it yourself to avoid multiple commits and pull requests to this repository at every change.
```

Comment:
```suggestion
However, if your Index Meta-Database is rapidly changing (e.g., because the list of sub-databases changes often), please host it yourself to avoid multiple commits and pull requests to this repository at every change.
```